zerver/webhooks/beeminder/view.py

```diff
@@ -48,7 +51,7 @@ def api_beeminder_webhook(request: HttpRequest, user_profile: UserProfile,
         # In this case PM will be sent to user
         p = get_user_profile_by_email(email)
         body = u"I am the Beeminder bot! :octopus: \n You are going to derail from \
-        goal **{}** in **{:0.1f} hours** \n You need **{}** to avoid derailing \n * Pledge: **{}$**{}"
+goal **{}** in **{:0.1f} hours** \n You need **{}** to avoid derailing \n * Pledge: **{}$**{}"
```

Comment:
A good idiom here to avoid the ugly line wrapping might be:

~~~ py
expected_body = '\n'.join([
   'I am the Beeminder bot!' :octopus:',
   # ...
])